packages/trader/src/App/Components/Form/CompositeCalendar/calenar-icon.jsx

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Icon } from '@deriv/components';
+
+/**
+ * Generate Calendar Icon and open
+ * related calendar by clicking on it
+ */
+const CalendarIcon = ({ onClick }) => <Icon onClick={onClick} icon='IcCalendarDatefrom' className='inline-icon' />;
+
+CalendarIcon.propTypes = {
+    onClick: PropTypes.func, // Function that opens a calendar
```

Comment:
this comment too - doesn't necessarily have to be a function that opens a calendar

Reply:
Thanks @oskar-binary you're right ✋, from now I'll use comments for complicated codes only.